drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/plugins/ConditionColumnPlugin.java

```diff
@@ -199,7 +199,7 @@ void prepareValues() {
             editingCol().setOperator(null);
         }
 
-        if (constraintValue() != BaseSingleFieldConstraint.TYPE_LITERAL) {
+        if (!(tableFormat() == GuidedDecisionTable52.TableFormat.LIMITED_ENTRY || constraintValue() == BaseSingleFieldConstraint.TYPE_LITERAL)) {
```

Comment:
Thank you for the idea, @jomarko. I did something a little bit different, but I kept the idea of DRYing the code ;-)